src/share/timing/gptl.c

```diff
@@ -5363,4 +5889,15 @@ static void print_threadmapping (FILE *fp)
   fprintf (fp, "threadid[0] = 0\n");
 }
 
+/*
+** serial_region: determine whether in a serial or parallel region
+**
+** Return value: true (1) or false (0)
+*/
+
+static int serial_region ()
+{
+  return 1;
+}
+
 #endif
```

Comment:
I guess my question about the gptl changes is - are we getting closer to the gptl git code or are we diverging further?

Reply:
@worleyph ?

Reply:
Diverging further, but talk to @rljacob . At Rob's request, I went through and added all of the timing library customizations back into the February 2018 version of GPTL . (I couldn't build the most recent version, despite some suggestions from Jim Rosinski, but there were almost no source changes since February - just moving to autoconf and reorganizing the directory structure.) I won't be able to submit PRs to the GPTL git repository until someone can hold my hand and show me how to build the most recent version. I also do not know how to modify CIME to use the public release of GPTL, which is a requirement for me to test this there. I also do not know if Jim Rosinski will be willing to accept the modifications (though they are almost all just new options, and are disabled by default unless using perf_mod.F90). Jim R. doesn't know either, since he hasn't seen any PRs yet :-).

Reply:
@jedwards4b these are closer enough.

Reply:
I'll accept it.   I'll try to make a cime branch with the latest gptl build for @worleyph